core/src/test/java/greencity/controller/SuperAdminControllerTest.java

```diff
@@ -127,43 +129,192 @@ void editTariffService() throws Exception {
     }
 
     @Test
-    void editService() throws Exception {
-        EditServiceDto dto = ModelUtils.getEditServiceDto();
-        ObjectMapper objectMapper = new ObjectMapper();
-        String ServiceResponseDtoJSON = objectMapper.writeValueAsString(dto);
-        mockMvc.perform(put(ubsLink + "/editService/" + 1L)
-            .content(ServiceResponseDtoJSON)
-            .contentType(MediaType.APPLICATION_JSON)
-            .principal(principal))
-            .andExpect(status().isOk());
+    void createService() throws Exception {
+        CreateServiceDto dto = ModelUtils.createServiceDto();
+        String requestedJson = new ObjectMapper().writeValueAsString(dto);
+
+        mockMvc.perform(post(ubsLink + "/createService")
+            .principal(principal)
+            .content(requestedJson)
+            .contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isCreated());
```

Comment:
could you please check that `superAdminService.addService()` is called with expected params?

Reply:
checked